Tensile/Common.py

```diff
@@ -1339,12 +1350,56 @@ def gfxArch(name):
 def gfxName(arch):
     return 'gfx' + ''.join(map(str,arch))
 
+def restoreDefaultGlobalParameters():
+  """
+  Restores `globalParameters` back to defaults.
+  """
+  global globalParameters
+  global defaultGlobalParameters
+  # Can't just assign globalParameters = deepcopy(defaultGlobalParameters) because that would
+  # result in dangling references, specifically in Tensile.Tensile().
+  globalParameters.clear()
+  for key, value in deepcopy(defaultGlobalParameters).items():
+    globalParameters[key] = value
+
+def printTable(rows):
+  rows = list([[str(cell) for cell in row] for row in rows])
+  colWidths = list([max([len(cell) for cell in col]) for col in zip(*rows)])
+
+  for row in rows:
+    for (width, cell) in zip(colWidths, row):
+      pad = ' ' * (width - len(cell))
+      print(pad, cell, sep='', end=' ')
+    print()
+
+def printCapTable(parameters):
+  import itertools
+  archs = [(0,0,0)] + parameters["SupportedISA"]
+  gfxNames = list(map(gfxName, archs))
+
+  headerRow = ['cap'] + gfxNames
+
+  def capRow(caps, cap):
+    return [cap] + [('1' if cap in caps[arch] and caps[arch][cap] else '0') for arch in archs]
+
+  allAsmCaps = set(itertools.chain(*[caps.keys() for arch, caps in parameters["AsmCaps"].items()]))
+  allAsmCaps = sorted(allAsmCaps)
+  asmCapRows = [capRow(parameters["AsmCaps"], cap) for cap in allAsmCaps]
+
+  allArchCaps = set(itertools.chain(*[caps.keys() for arch, caps in parameters["ArchCaps"].items()]))
+  allArchCaps = sorted(allArchCaps)
+  archCapRows = [capRow(parameters["ArchCaps"], cap) for cap in allArchCaps]
+
+  printTable([headerRow] + asmCapRows + archCapRows)
+
 ################################################################################
-# Assign Global Parameters
-# each global parameter has a default parameter, and the user
-# can override them, those overridings happen here
 ################################################################################
 def assignGlobalParameters( config ):
+  """
+  Assign Global Parameters
+  Each global parameter has a default parameter, and the user
+  can override them, those overridings happen here
+  """
 
   global globalParameters
 
```

Comment:
@ Line 1409 - Message about Benchmark.yaml might be confusing here as we might not always use this function in that context.